net-mgmt/net-snmp/src/opnsense/mvc/app/models/OPNsense/Netsnmp/General.xml

```diff
@@ -19,5 +19,9 @@
             <default></default>
             <Required>N</Required>
         </syscontact>
+        <listen type="CSVListField">
+            <default></default>
+            <Required>N</Required>
```

Comment:
This should have at least some basic validation for IP addresses for additional safety

Reply:
You remeber that it's still not possible to mask for v4 and v6? Thats why Ad created the HostnameField or v6 and v4 values for NetworkField.

Reply:
Some plugins use extended regex validation instead. Let me see if we can’t have a multiple field with IPv4 and IPv6... :)

Reply:
Doesn't this do what we want? https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/models/OPNsense/TrafficShaper/TrafficShaper.xml#L239-L243

PS: you can remove the empty default...

Reply:
Oh, it's really that easy? 👍 I tested it on latest master also, but an empty field get's value "undefined" by default, when I check save the Validator for NetworkField tells me the input is wrong ;)

Reply:
It's used in IPFW  shaper, the default seems to be "any" instead of empty, but it should virtually be the same for the validator CC @AdSchellevis 

we can probably also make these fields tokenizers very easily, in fact that would be a good switch for the shaper fields in core to make the feature more obvious.

Reply:
We should indeed switch the src/dst shaper fields to tokenizers for clearity, let me fix that.